core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -29,6 +29,13 @@ export class AdminRouterService {
   currentTabHash: string = (
     AdminPageConstants.ADMIN_TAB_URLS.ACTIVITIES);
 
+  /**
+   * Iterates through the ADMIN_TAB_URLS map and returns the
+   * tab name corresponding to the hash.
+   * @param {string} tabHash The string after the '#' character in the URL.
+   * @returns {string|null} The corresponding tab name, or null
```

Comment:
Changed as needed

Reply:
Hi @anumehaagrawal -- just a note, you don't need to write this. Replying "Done" to the original conversation thread is enough. That helps keep all the conversation in a single place.

Thanks!